ruler-gradle-plugin/src/main/kotlin/com/spotify/ruler/plugin/dependency/EntryParser.kt

```diff
@@ -0,0 +1,85 @@
+/*
+* Copyright 2021 Spotify AB
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package com.spotify.ruler.plugin.dependency
+
+import com.spotify.ruler_cli.dependency.ArtifactResult
+import com.spotify.ruler_cli.dependency.DependencyEntry
+import com.spotify.ruler_cli.dependency.DependencyParser
+import com.spotify.ruler_cli.models.AppInfo
+import org.gradle.api.Project
+import org.gradle.api.artifacts.Configuration
+import org.gradle.api.artifacts.result.ResolvedArtifactResult
+import org.gradle.api.attributes.Attribute
+import java.io.File
+
+/** Responsible for parsing and extracting entries from dependencies. */
+class EntryParser {
```

Comment:
Wouldn't keep this class at all, thinking of moving the logic inside the `RulerTask` class